src/layout/Footer/Footer.tsx

```diff
@@ -1,9 +1,77 @@
-import { FooterWrapper } from './Footer.styled';
+import { AppBar, Button, Menu, MenuItem, Typography } from '@mui/material';
+import { StyledToolBar } from './Footer.styled';
+import LanguageRoundedIcon from '@mui/icons-material/LanguageRounded';
+import { useState } from 'react';
+
+const languages: string[] = ['English - en', 'Espanol - es'];
 
 export const Footer = () => {
+    const [selectedLanguage, setSelectedLanguage] = useState('English - en');
```

Comment:
better to use some enum

Reply:
This will be replaced with redux later, so I just put this as placeholder till then.